infrastructure/protocol-upgrade/src/hyperchain-upgrade.ts

```diff
@@ -66,7 +84,22 @@ async function hyperchainUpgrade2() {
     process.chdir(cwd);
 }
 
+async function hyperchainUpgrade3() {
+    const cwd = process.cwd();
+    process.chdir(`${process.env.ZKSYNC_HOME}/contracts/l1-contracts/`);
+
+    await spawn(`yarn hyperchain-upgrade-3 | tee deployL1.log`);
+    process.chdir(cwd);
+}
+
 async function preparePostUpgradeCalldata() {
+    console.log(
+        'Preparing post upgrade calldata',
```

Comment:
I would suggest to either remove the log or make sure that each outputted value has corresponding text before it so it is easier to understand without looking into the code